pkg/operator/controller/garden/care/add.go

```diff
@@ -94,7 +94,7 @@ func (r *Reconciler) AddToManager(ctx context.Context, mgr manager.Manager) erro
 
 // GardenPredicate is a predicate which returns 'true' for create events, and for update events in case the garden was
 // successfully reconciled.
-func (r *Reconciler) GardenPredicate() predicate.Predicate {
+func GardenPredicate() predicate.Predicate {
```

Comment:
Please move this function to `pkg/operator/predicate` when you use it in multiple controllers.